### DISCUSSION TITLE
No value is resolved for given XPath

### DISCUSSION THREAD

**Discussion by glennbraganza** [Huachao/vscode-restclient#768] [Q&A] [answered]
I can't seem to resolve the 'No value is resolved for given XPath' error when using the plugin and calling an API with an XML response. Here's an example

```
### Test API 1
# @name getspace
GET https://www.demo.com/v1/spaces
Authorization: Bearer {{BEARER_TOKEN}}
Content-Type: application/xml
```

This is a sample Response

```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<SummaryResponse 
  xmlns="http://www.demo.com/schema/1/">
  <spaceId>A2114</spaceId>
  <Role>Gardener</Role>
  <Jurisdiction>Garden</Jurisdiction>
  <Status>Active</Status>
</SummaryResponse> 

```
I just want to get the value of the spaceId field and pass it to the next call. But I get the error in VSCode when I highlight the **getspace.response.body.//SummaryResponse/spaceId** variable

```
### Test API 2
GET https://{{API_SERVER_URL}}/v1/space?spaceId={{getspace.response.body.//SummaryResponse/spaceId}}
Authorization: Bearer {{BEARER_TOKEN}}
```

Can't figure out what I'm doing wrong. Appreciate any help

**Comment by glennbraganza** [comment, marked as answer]
Found the answer after searching in the issues tabs:

This works for me
GET https://{{API_SERVER_URL}}/v1/space?spaceId={{getspace.response.body./*[local-name()='SummaryResponse']//*[local-name()='spaceId']}}
Authorization: Bearer {{BEARER_TOKEN}}

**Reply by Huachao** [reply]
That's the work around for now
